packages/auth-browser/src/lib/chains/eth.ts

```diff
@@ -726,7 +730,8 @@ export const signAndSaveAuthMessage = async ({
   resources,
   expiration,
   uri,
-}: signAndSaveAuthParams): Promise<AuthSig> => {
+}: signAndSaveAuthParams,
```

Comment:
not sure why we need to inject the client instance. cant the sdk and nodes independently query the block height? requiring a client instance for this method is a bit of a heavy lift and does not seem needed for simply querying the block height has this can be done with a scoped rpc client in `signAndSaveAuthMessage`